package.json

```
@@ -0,0 +1,24 @@
+{
+    "devDependencies": {
+        "@symfony/webpack-encore": "^0.31.0",
+        "@vue/babel-helper-vue-jsx-merge-props": "^1.2.1",
+        "@vue/babel-preset-jsx": "^1.2.2",
+        "core-js": "^3.0.0",
```

```suggestion
        "core-js": "^3.0.0",
        "css-loader": "^5.0.0",
```
To resolve `warning " > vue-loader@15.9.3" has unmet peer dependency "css-loader@*".`